code/modules/reagents/reagent_containers/drinks/juicebox.dm

```diff
@@ -101,7 +101,7 @@
 	desc = "Juice in a box; who knows what flavor!"
 
 /obj/item/chems/drinks/juicebox/sensible_random/proc/juice_it()
-	var/drinktypes = subtypesof(/decl/material/liquid/drink/juice)
+	var/list/drinktypes = decls_repository.get_decls_of_subtype(/decl/material/liquid/drink/juice)
```

Comment:
This should be `get_decl_paths_of_subtype`, since as far as I can tell, add_reagent takes a typepath and not a decl instance.